julianduss.pub

```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGzuHbqqigfERZvbdeSnLIsHzWd3TfD0yjvsoVijjrnM 
```

Comment:
Common name missing

Reply:
Whats that?

Reply:
You should providen your email address or your name for easier administration. See https://www.man7.org/linux/man-pages/man1/ssh-keygen.1.html#:~:text=will%20be%20ignored.-,%2DC%20comment,-Provides%20a%20new